frontend/app/src/composables/filter-paginate.ts

```diff
@@ -12,6 +12,7 @@ import {
   type LocationQuery,
   RouterPaginationOptionsSchema
 } from '@/types/route';
+import { getUpdatedKey } from '@/services/axios-tranformers';
```

Comment:
At some point, we should maybe move some of the functions since this is clearly not used only in the Axios transformers. Don't do it now but we need to keep that in mind.